File: depth/32_c_lib/src/main.zig
const std = @import("std");
const math = @import("math");

pub fn main() !void {
    const a = 21;
    const b = 21;
    const c = math.add(a, b);

    std.debug.print("declaration lib zig: a + b == {}\n", .{c});
    std.debug.print("declaration lib zig: a++ == {}\n", .{math.increment(a)});
}

File: depth/32_c_lib/src/main.c
#include <stdio.h>

#include "math.h"

int main(void) {
	int a = 21;
	int b = 21;
	int c = add(a, b);

	printf("c: a + b == %d\n", c);
	printf("c: a++ == %d\n", increment(a));

	return 0;
}
